### PR TITLE
Qingying redlands recommendations

### LINKED ISSUE #19: Add Redlands Redlands file
- [ ]  Create a branch
- [ ]  Add file/commits
- [ ]  Open a PR
- [ ]  Have a discussion about the PR (COLLABORATE)
- [ ]  Make edits
- [ ]  Get: :+1: 
- [ ]  Merge PR

### PR DESCRIPTION
**Pull request by redlands**
closes #19 

hey @YuanZeng, does this look like a good list of recommendations? 
:

cc:  @crichID

### CODE REVIEW

File: Qingying-Redlands.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#Redlands, CA
+
+
+
+#Places to Eat
+City Tacos
+
+
+#Places to Go
+Sea World
```
Comment (YuanZeng) [inline Qingying-Redlands.md:12]: Sea World move to redlands now :smile: ?

**Comment by YuanZeng** [comment]
Any recommendations for places around Redlands to go for a short vacation?

File: Qingying-Redlands.md (hunk 1 above)
Comment (redlands) [reply Qingying-Redlands.md:12]: right, i am dreaming in San Diego, :smile:

**Comment by redlands** [comment]
@YuanZeng  Added brweries, does this good good to :shipit: ?

**Pull request merged**